cli/commands/project.go

```diff
@@ -29,21 +29,23 @@ func (cmd *Project) Commands() []cli.Command {
 		Before:      cmd.Before,
 	}
 
-	if subcommands := cmd.GetScriptsAsSubcommands(); subcommands != nil {
-		command.Subcommands = subcommands
-	}
-
 	create := ProjectCreate{}
 	command.Subcommands = append(command.Subcommands, create.Commands()...)
```

Comment:
Since it is not a pointer to a slice, it gets initialized to an empty slice by default.

https://github.com/urfave/cli/blob/master/command.go#L44